tools/swift-inspect/Sources/swift-inspect/Inspector.swift

```diff
@@ -67,11 +67,19 @@ class Inspector {
   }
   
   func getAddr(symbolName: String) -> swift_addr_t {
-    let symbol = CSSymbolOwnerGetSymbolWithMangledName(swiftCore, symbolName)
+    let symbol = CSSymbolOwnerGetSymbolWithMangledName(swiftCore,
+                                                       "_" + symbolName)
```

Comment:
Doesn't this need to take into account the platform before adding a user label prefix?  Linux, android, Windows x86_64 and ARM do not use `_` (Windows x86, MIPS, and Darwin do).

Reply:
This is Darwin-specific code, as the `CS*` calls are only available there. Support for other OSes will need a new `Inspector` implementation that uses the appropriate calls for that OS.

Reply:
Fair enough.  Though, it raises the question of whether this should be inside `CS*` or outside still.